src/styles/block/page.scss

```diff
@@ -0,0 +1,8 @@
+.page {
+  font-family: Roboto, sans-serif;
+  color: $main-color;
```

Comment:
The variable `$main-color` is used here. Ensure that this variable is defined in your SCSS files. If it's not defined, it will cause a compilation error.